### PR TITLE
CLOUD-677 Update README.md, other quick fixes

### PR DESCRIPTION
**Pull request by spron-in**
[CLOUD-677](https://jira.percona.com/browse/CLOUD-677) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

Correct the naming
Add help
Simplify quick installation

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| storage | passed |
| big-data | passed |
| haproxy | passed |
| scheduled-backup | passed |
| scaling | passed |
| recreate | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade-consistency | passed |
| users | failed |
| demand-backup-encrypted-with-tls | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1097/commits/089ec4a2efcdbf1d12d9113c94274bb26a31dd8d
image: `perconalab/percona-xtradb-cluster-operator:PR-1097-089ec4a2`

**Review by fiowro — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade-consistency | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1097/commits/b1dba85e328ef78ea6b7f52785b4a10a8e33e427
image: `perconalab/percona-xtradb-cluster-operator:PR-1097-b1dba85e`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| big-data | passed |
| haproxy | passed |
| scaling | passed |
| scheduled-backup | passed |
| storage | passed |
| recreate | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| affinity | passed |
| one-pod | passed |
| pitr | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| upgrade-consistency | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1097/commits/d1c941f40e973fa039da72c0989428dd2b9eb23f
image: `perconalab/percona-xtradb-cluster-operator:PR-1097-d1c941f4`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| scheduled-backup | passed |
| upgrade-haproxy | passed |
| storage | passed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| init-deploy | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| upgrade-consistency | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1097/commits/2ad191539e577fa038c09336469da5762a0c07e3
image: `perconalab/percona-xtradb-cluster-operator:PR-1097-2ad19153`

File: README.md [hunk 1]
```diff
@@ -20,12 +20,43 @@ Percona Operators are based on the [Operator SDK](https://github.com/operator-fr
...
+
+It usually takes two steps to deploy Percona XtraDB Cluster on Kubernetes.
+
+Deploy the Operator from `deploy/bundle.yaml`:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/main/deploy/bundle.yaml
+```
+
+Deploy the database cluster itself from `deploy/cr.yaml`:
```
Comment (hors) [inline README.md:54]: `deploy/cr.yaml or deploy/cr-minimal.yaml `?

**Review by hors — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (spron-in) [reply README.md:54]: Let's leave cr.yaml here.

**Review by fiowro — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | failed |
| haproxy | failed |
| big-data | failed |
| scaling | failed |
| upgrade-haproxy | failed |
| storage | failed |
| scheduled-backup | failed |
| restore-to-encrypted-cluster | failed |
| init-deploy | failed |
| scaling-proxysql | failed |
| self-healing | passed |
| self-healing-chaos | failed |
| upgrade-proxysql | failed |
| demand-backup | failed |
| limits | failed |
| security-context | failed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | failed |
| smart-update | failed |
| pitr | failed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| upgrade-consistency | failed |
| affinity | failed |
| demand-backup-encrypted-with-tls | failed |
| one-pod | failed |
| auto-tuning | failed |
| proxysql-sidecar-res-limits | failed |
| users | failed |
| tls-issue-self | failed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |
| validation-hook | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1097/commits/094c290ebac51dc0c6ba1dc3320e2dac53eaeda9
image: `perconalab/percona-xtradb-cluster-operator:PR-1097-094c290e`

**Review by hors — APPROVED** [review]

**Pull request merged**